source/frontend/src/features/mapSideBar/acquisition/tabs/stakeholders/detail/StakeHolderView.test.tsx

```diff
@@ -128,11 +150,13 @@ describe('StakeHolderView component', () => {
       getMockApiInterestHolders()[0].interestHolderProperties[0],
     );
 
+    organizerMock.getInterestProperties.mockReturnValue([model]);
+
     model.identifier = 'PID: 025-196-375';
     const { getByText } = setup({
       props: {
-        groupedInterestProperties: [],
-        groupedNonInterestProperties: [model],
+        //groupedInterestProperties: [],
```

Comment:
? if this is a wip, please fix. if not, can you remove these comments?

Reply:
ups. forgot to cleanup this tests